plugins/hls-fourmolu-plugin/hls-fourmolu-plugin.cabal

```diff
@@ -9,21 +9,27 @@ license:            Apache-2.0
 license-file:       LICENSE
 author:             The Haskell IDE Team
 copyright:          The Haskell IDE Team
+homepage:           https://github.com/haskell/haskell-language-server
+bug-reports:        https://github.com/haskell/haskell-language-server/issues
```

Comment:
It took me a minute to figure out the Git repository for this package, so I added these fields to the `.cabal` file. Technically an unrelated change, would be easy to put somewhere else.